### PR TITLE
TE-1348 - add test_verify_removed_access test case and methods

### PR DESCRIPTION
**Pull request by jordan-mcgillivray005**

### CODE REVIEW

**Comment by jordan-mcgillivray005** [comment]
Only one run at the moment due to the regression running. 
image

File: keywords/webui/permissions.py [hunk 1]
```diff
@@ -18,15 +18,22 @@ def assert_dataset_execute_access(cls, pool: str, dataset: str, username: str, p
...
+        # cr_dir = f"{username}exec_dir"
+        # command = f'cd /mnt/{pool}/test ; chmod 777 . ; touch {file} ;  echo -n "mkdir /mnt/{pool}/{dataset}/{cr_dir}" | cat > {file} ; chmod 777 {file}'
+        # SSH.get_output_from_ssh(command, private_config['IP'], private_config['USERNAME'], private_config['PASSWORD'])
+        # command2 = f"cd /mnt/{pool}/test ; ./{file}"
+        # SSH.get_output_from_ssh(command2, private_config['IP'], username, password)
+        # command3 = f"cd /mnt/{pool} ; sudo ls -al {dataset}"
+        # value = SSH.get_output_from_ssh(command3, private_config['IP'], private_config['USERNAME'], private_config['PASSWORD'])
+        # return cr_dir in value.stdout.lower()
+        command = f"cd /mnt/{pool}/{dataset}"
+        value = SSH.get_output_from_ssh(command, private_config['IP'], username, password)
```
Comment (ericbsd) [inline keywords/webui/permissions.py:32]: This should be under keyword/ssh/permissions.

File: keywords/webui/permissions.py [hunk 2]
```diff
@@ -104,6 +111,28 @@ def assert_dataset_owner(cls, name: str) -> bool:
...
+        """
+        command = f"sudo rm -rf /mnt/{pool}/{dataset}/unix_test_ownerfile.txt"
+        value = SSH.get_output_from_ssh(command, private_config['IP'], private_config['USERNAME'], private_config['PASSWORD'])
+        command2 = f"sudo rm -rf /mnt/{pool}/{dataset}/unix_test_groupuserfile.txt"
+        value2 = SSH.get_output_from_ssh(command2, private_config['IP'], private_config['USERNAME'], private_config['PASSWORD'])
+        command3 = f"sudo rm -rf /mnt/{pool}/{dataset}/unix_test_otheruserfile.txt"
+        value3 = SSH.get_output_from_ssh(command3, private_config['IP'], private_config['USERNAME'], private_config['PASSWORD'])
+        command4 = f"cd /mnt/{pool} ; sudo ls -al {dataset}"
+        SSH.get_output_from_ssh(command4, private_config['IP'], private_config['USERNAME'], private_config['PASSWORD'])
+        assert value.status is True and value2.status is True and value3.status is True
```
Comment (ericbsd) [inline keywords/webui/permissions.py:134]: Same here this should be under keyword/ssh/permissions.

**Review by ericbsd — APPROVED** [review]

File: keywords/webui/permissions.py (hunk 1 above)
Comment (jordan-mcgillivray005) [reply keywords/webui/permissions.py]: Fixed.

File: keywords/webui/permissions.py (hunk 2 above)
Comment (jordan-mcgillivray005) [reply keywords/webui/permissions.py]: Fixed.

**Review by ericbsd — APPROVED** [review]
LGTM

**Pull request merged**